cxtool/cx-integrations.sh

```diff
@@ -0,0 +1,454 @@
+#!/bin/bash
+
+generate=false
+deploy=false
+namespace=default
+endpoint=api.eu2.coralogix.com
+appdynamic=true
+subdynamic=true
+platform=kubernetes
+outputdir=output
+destination=local
+
+function main {
+
+  function upload {
+    if [ "$destination" == "s3" ]; then
+      if [ -z "$bucket" ]; then echo "bucket name is empty!"; exit 1; fi
+      if [ -z "$bucketpath" ]; then 
+        aws s3 mv ./$outputdir/$integration-manifests.yaml s3://$bucket/$integration-manifests.yaml 2>&1
+        if [ $? -ne 0 ]; then 
+          exit 1
+        fi
+      else
+        aws s3 mv ./$outputdir/$integration-manifests.yaml s3://$bucket/$bucketpath/$integration-manifests.yaml 2>&1
+        if [ $? -ne 0 ]; then 
+          exit 1
+        fi      fi
+    
+    elif [ "$destination" == "github" ]; then
+      if [ -z "$giturl" ]; then echo "git url is empty!"; exit 1; fi
+      git clone $giturl 2>&1
+      if [ $? -ne 0 ]; then exit 1; fi
+      git add ./$outputdir/$integration-manifests.yaml
+      git commit -m "Adding fluent-bit-http kubernetes manifests"
+      git push 2>&1
+      if [ $? -ne 0 ]; then exit 1; fi
+      rm -rf ./$outputdir
+
+    elif [ "$destination" == "local" ]; then
+      if [ ! -z "$path" ]; then
+        mv ./$outputdir/$integration-manifests.yaml $path 2>&1
+        if [ $? -ne 0 ]; then exit 1; fi
+        rm -rf ./$outputdir
```

Comment:
I prefer to remove this functionality of deletion, by an easy mistake it could remove entire someone computer / or instance